use-timescale/compression/compression-policy.md

```diff
@@ -59,6 +59,20 @@ SELECT * FROM timescaledb_information.jobs
 
 For more information, see the API reference for [`timescaledb_information.jobs`][timescaledb_information-jobs].
 
+## Pause compression policy
+
+To disable a compression policy temporarily:
```

Comment:
We should probably tell people how to find the jobid here.

SELECT * FROM timescaledb_information.jobs where proc_name = 'policy_compression' AND relname = '<hypertablename>'